packages/web/src/components/Speakers.js

```diff
@@ -5,18 +5,20 @@ import Flippedmb from "./Flippedmb";
 const Speakers = () => {
     return (
    
-    <div className="Speaker" hidden>     
+    <div className="Speaker">     
     <div className="SpeakerDesc"> 
     <h1 className="heading" id="Speakers">Speakers</h1>
         <Flippedmb/>   
         
         <div className="gallery">
-      <img src="https://cdn.freecodecamp.org/curriculum/css-photo-gallery/1.jpg"></img>
-      <img src="https://cdn.freecodecamp.org/curriculum/css-photo-gallery/2.jpg"></img>
-      <img src="https://cdn.freecodecamp.org/curriculum/css-photo-gallery/3.jpg"></img>
-      <img src="https://cdn.freecodecamp.org/curriculum/css-photo-gallery/4.jpg"></img>
-      <img src="https://cdn.freecodecamp.org/curriculum/css-photo-gallery/5.jpg"></img>
-      <img src="https://cdn.freecodecamp.org/curriculum/css-photo-gallery/6.jpg"></img>
+      <img src="https://media.discordapp.net/attachments/756143084189778043/1039950769203466271/boris_angelov_name.png?width=695&height=707"></img>
```

Comment:
Can you add the images to the public folder and reference them from there?

Reply:
bruh